client/src/data_layer_configs/lead_service_lines_by_water_systems_config.ts

```diff
@@ -40,19 +51,19 @@ const leadConnectionLegendInterpolation = [
   ['linear'],
   // Provides ratio of lead service lines : total service lines.
   ['/', ['get', 'lead_connections_count'], ['get', 'service_connections_count']],
-  ...LEGEND_COLOR_MAPPING,
+  ...INTERPOLATION_COLOR_MAPPING,
 ];
 
 const legendInfo: LegendInfo = {
-  title: 'Proportion of lead lines to all service lines',
+  title: 'Percentage of lead service lines',
   bucketMap: colorMapToBuckets(LEGEND_COLOR_MAPPING),
 };
 
 export const styleLayer: FillLayer = {
   id: `${MapLayer.LeadServiceLineByWaterSystem}-style`,
   source: MapLayer.LeadServiceLineByWaterSystem,
   // Corresponds to the table in the database.
-  'source-layer': 'public.water_systems',
+  'source-layer': 'public.lead_connections_function_source',
```

Comment:
nit: Make a const for table name just like other data layers use here and in tile url

Reply:
Done